velox/expression/tests/ExpressionFuzzerTest.cpp

```diff
@@ -57,7 +57,11 @@ int main(int argc, char** argv) {
       // cardinality passing a VARBINARY (since HLL's implementation uses an
       // alias to VARBINARY).
       "cardinality",
-  };
-  size_t initialSeed = FLAGS_seed == 0 ? std::time(nullptr) : FLAGS_seed;
-  return FuzzerRunner::run(FLAGS_only, FLAGS_steps, initialSeed, skipFunctions);
+      // Decimal functions require variable bindings (tryBind) to be executed
```

Comment:
Skipping "plus" would reduce coverage. Can we instead add some logic to the Fuzzer to skip function signatures that use decimal types?

Reply:
Can we work this in a future PR built on top of Addition PR.

Reply:
Addressed here: https://github.com/facebookincubator/velox/pull/2076